app/javascript/packages/multiselect-dropdown/package.json

```diff
@@ -37,13 +37,13 @@
     "fuse.js": "^6.4.6",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "@rescript/react": "^0.10.1"
+    "@rescript/react": "^0.10.2"
   },
   "scripts": {
     "dev": "parcel index.html",
     "build": "parcel build index.html",
     "test": "echo \"Error: run tests from root\" && exit 1",
     "re:build": "bsb -make-world -clean-world",
```

Comment:
Shouldn't this `re:build` command command change as well? I noticed that the main `package.json` has been updated.

This applies to other `package.json` files in the other libs as well.